docs/rules/multi-word-component-names.md

```diff
@@ -16,7 +16,7 @@ since: v7.20.0
 This rule require component names to be always multi-word, except for root `App`
 components, and built-in components provided by Vue, such as `<transition>` or
 `<component>`. This prevents conflicts with existing and future HTML elements,
-since all HTML elements are a single word.
+since every HTML element is a single word.
```

Comment:
I think this sounds odd and incorrect now. Let's make "single word" plural in the original sentence, then it should be fine, right?

```suggestion
since all HTML elements are single words.
```